lib/ansible/modules/cloud/openstack/os_network.py

```diff
@@ -69,7 +69,16 @@
    availability_zone:
      description:
        - Ignored. Present for backwards compatibility
-requirements: ["openstacksdk"]
+   port_security_enabled:
+     description:
+        -  Whether port security is enabled on the network or not.
+           Network will use OpenStack defaults if this option is
+           not utilised.
+     type: bool
+     version_added: "2.7"
+requirements:
+     - "openstacksdk"
+     - "shade >= 1.27.1"
```

Comment:
Please remove the shade dependency, it is no longer required.

Reply:
done.